test_pool/power_wakeup/operating_system/test_os_u004.c

```diff
@@ -93,6 +89,9 @@ payload4()
 
```

Comment:
@NicholasGraves Can you change the timer_expire_val = val_get_counter_frequency() * g_wakeup_timeout and test if the wakeup set failsafe still required 1.5 times the WDT.

In general for BSA wakeup tests, the failsafe interrupt timers have twice the timeout value of timer under test.